docs/en_US/jailbreak/installing-electra.md

```diff
@@ -3,9 +3,6 @@ lang: en_US
 title: Installing Electra
 description: Guide to installing Electra
 permalink: /installing-electra
-redirect_from:
-  - /electra
-  - /el
```

Comment:
why was this removed?

Reply:
that was a mistake I meant to remove it on installing-electratv.md